### ISSUE TITLE
[docs] Contradicting migration docs

### ISSUE THREAD

**Issue by jagomf** [flutter/flutter#98837]
When following docs to [migrate from Android Embedding V1 to V2](https://github.com/flutter/flutter/wiki/Upgrading-pre-1.12-Android-projects#full-flutter-app-migration), in step 4, after setting both `LaunchTheme` and `NormalTheme` styles, next thing to do is *"Configure `MainActivity` to start with launch theme (...)"*. One of the specified things you have to do is make sure you have a `<meta-data android:name="io.flutter.embedding.android.SplashScreenDrawable">`.

However, after setting this, when you build your app you get a deprecation warning:

    W/FlutterActivityAndFragmentDelegate(14336): A splash screen was provided to Flutter, but this is deprecated. See flutter.dev/go/android-splash-migration for migration steps.

When visiting [mentioned URL](https://flutter.dev/go/android-splash-migration), you can see that `io.flutter.embedding.android.SplashScreenDrawable` **is deprecated and should actually be removed**, which is the opposite you get told to do in previous docs.

Further than this, at least in my case, removing the aforementioned `<meta-data>` causes splash screen not to show the app logo anymore.

---

**TL;DR** Two docs telling to do contradicting stuff.

**Comment by darshankawar** [comment]
@jagomf 
I think your report is same as mentioned in this similar open issue: https://github.com/flutter/flutter/issues/97859, so please follow-up in it for further updates.
Closing this as a duplicate. If you disagree, write in comments and I'll reopen it.
